### ISSUE TITLE
Status message not announced

### ISSUE THREAD

**Issue by MajoMaldonado** [equalizedigital/boardscribe#117]
Location / URL: https://unassuming-eland-bd9068.instawp.co/wp-admin/edit.php?post_type=edbs_meeting&page=edbs-settings&tab=import&edbs_import_success=3&edbs_import_skipped=0 

Screenshot: 

<img width="747" height="362" alt="Image" src="https://github.com/user-attachments/assets/8625f7c7-65a8-4ebc-af5c-453b459599f6" />

WCAG S.C  4.1.3 Status Messages / Best Practice 

Issue Description: After importing a CSV file, the page reloads and displays the confirmation message “Import complete. 3 rows imported, 0 skipped.” The confirmation is not automatically announced by the screen reader, so screen reader users may not immediately know whether the import was completed successfully.
